packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/billing-tile/BillingTile.component.tsx

```diff
@@ -27,70 +30,93 @@ type TBillingTileProps = {
+                  value: isLoading ? (
+                    <OsdsSkeleton />
+                  ) : (
+                    <Description>{user?.email}</Description>
+                  ),
+                },
+                {
+                  id: 'serviceRenew',
+                  label: t('managed_vcd_dashboard_service_renew'),
+                  value: nextBillingDate ? (
+                    <Links
+                      onClickReturn={() => {}}
+                      label={dateTimeFormat?.format(new Date(nextBillingDate))}
                     />
-                    <OsdsTooltipContent
-                      slot="tooltip-content"
-                      className="break-normal"
-                    >
-                      {t('managed_vcd_dashboard_password_tooltip')}
-                    </OsdsTooltipContent>
-                  </OsdsTooltip>
-                </div>
-                <OsdsChip
-                  inline
-                  color={ODS_THEME_COLOR_INTENT.primary}
-                  className="ml-3"
-                  size={ODS_CHIP_SIZE.sm}
-                >
-                  {t('managed_vcd_dashboard_coming_soon')}
-                </OsdsChip>
-              </div>
-            ),
-          },
-        ]}
+                  ) : (
+                    <span>-</span>
+                  ),
+                },
+                {
+                  id: 'password',
+                  label: t('managed_vcd_dashboard_password'),
+                  value: isLoading ? (
+                    <OsdsSkeleton />
+                  ) : (
+                    <div className="flex-wrap">
+                      <div className="flex items-center gap-x-2">
+                        <OsdsLink disabled>
+                          {t('managed_vcd_dashboard_password_renew')}
+                        </OsdsLink>
+                        <OsdsTooltip className="flex items-center">
+                          <OsdsIcon
+                            className="cursor-pointer"
+                            name={ODS_ICON_NAME.HELP}
+                            size={ODS_ICON_SIZE.xxs}
+                            color={ODS_THEME_COLOR_INTENT.text}
+                          />
+                          <OsdsTooltipContent
+                            slot="tooltip-content"
+                            className="break-normal"
+                          >
+                            {t('managed_vcd_dashboard_password_tooltip')}
+                          </OsdsTooltipContent>
+                        </OsdsTooltip>
+                      </div>
+                      <OsdsChip
+                        inline
+                        color={ODS_THEME_COLOR_INTENT.primary}
+                        className="ml-3"
+                        size={ODS_CHIP_SIZE.sm}
+                      >
+                        {t('managed_vcd_dashboard_coming_soon')}
+                      </OsdsChip>
+                    </div>
```

Comment:
Why should we display Skeleton when isLoadingBillingServices while we have no data loading here ?